fuzz/README.md

```diff
@@ -104,12 +104,14 @@ FUZZ_FNAME: hfuzz_workspace/hashes_sha256/SIGABRT.PC.7ffff7c8abc7.STACK.18826d9b
 fff400610004
 ```
 
+If you don't see the summary because you fuzzed via `cargo`, run
+
+    checkReport <target>
+
 The final line is a hex-encoded version of the input that caused the crash. You
 can test this directly by editing the `duplicate_crash` test to copy/paste the
 hex output into the call to `extend_vec_from_hex`. Then run the test with
 
-    cargo test
-
-Note that if you set your `RUSTFLAGS` while fuzzing (see above) you must make
-sure they are set the same way when running `cargo test`.
+    RUSTFLAGS=--cfg=fuzzing cargo test
```

Comment:
In 3ea836c9c0615fbcf2beb48d64bc7c52518af452:

This is incorrect. The old text was correct.

Reply:
This has to do with the `fuzzing` feature gates on the unit tests. Those were introduced [in this commit](https://github.com/rust-bitcoin/rust-bitcoin/commit/64677282023295812c8b95212e0e5e96021040f2) to rust-bitcoin. Then [this second commit](https://github.com/rust-bitcoin/rust-bitcoin/commit/6649e1519305973a96d6d16e839711352c6044d8) added support for weak crypto where the `fuzzing` flag is disallowed. We don't have any of that in rust-simplicity, so we could stick with the original lines. If we want to be open to future changes, I would at least add a line that explains what the typical command in the typical scenario would looks like, so it's easy to copy and paste.